example.c

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+/* mraa header */
+#include "mraa/i2c.h"
+
+#include "fpga_mezz.h"
+
+int main()
+{
+    mraa_result_t status = MRAA_SUCCESS;
+    mraa_i2c_context i2c;
+
+    mraa_init();
+    i2c = mraa_i2c_init(I2C_BUS);
+
+    int input;
+
+    if (i2c == NULL) {
+        fprintf(stderr, "Failed to initialize I2C\n");
+        mraa_deinit();
+        return EXIT_FAILURE;
+    }
+
+    /* set slave address */
+    status = mraa_i2c_address(i2c, FPGA_MEZZI_ADDR);
+    if (status != MRAA_SUCCESS) {
+    }
+
+    u_int8_t ver;
+    ver = check_version(i2c);
```

Comment:
Since you are returning error case also, you should check it here.

Reply:
added